lib/install/validate/config_to_data.go

```diff
@@ -38,13 +39,8 @@ const (
 	httpsProxy = "HTTPS_PROXY"
 )
 
-// Finder is defined for easy to test
-type Finder interface {
-	ObjectReference(ctx context.Context, ref types.ManagedObjectReference) (object.Reference, error)
```

Comment:
Why not just replace the `ObjectReference` method here with the `Element` method you want to use?

It seems like both the old and new code use a `find.Finder` under the hood; the old code just refers to it via an interface (which generally seems like a good pattern, even if we're not taking full advantage of it).

Reply:
I'm confused why that's a good pattern?  In my mind it convolutes the meaning of Finder..now we have two...

Reply:
I think @zjs is proposing keeping the wrapping Finder interface so that we have the potential to mock it for testing.

Given this is one of the few pieces of vic code that vcsim is fully able to handle I'm inclined to use that as our "mock" instead.